docs/examples.md

```diff
@@ -180,5 +180,5 @@ systemd:
         WantedBy=multi-user.target
 ```
 
-[spec]: configuration-v1_0.md
+[spec]: specs.md
```

Comment:
`specs.md` is an empty document.  That's okay on the website, with index generation, but isn't great for anyone browsing docs in the GitHub UI.

Reply:
Indeed. But I don't have any idea to make it better. Manual indexing?

Reply:
I will extend the specs page to make it more interesting